Gemfile

```diff
@@ -17,6 +17,10 @@ group :development, :test do
   gem 'rubocop', require: false
 end
 
+group :test do
+  gem 'rails-controller-testing'
```

Comment:
I thought that it's only for the test so put it into group only for test instead of development and test.